notebooks/43_Decoding_SortedSpikes.ipynb

```diff
@@ -0,0 +1,1636 @@
+{
```

Comment:
<p>Should this pull data from a different sorter than <code>clusterless</code> (e.g. mountainsort4)? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/spyglass/pull/731/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Yes in general. I just didn't want to spend the time to run something through mountainsort.